### PR TITLE
Create doc for inline edit comments

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

File: ref/general/threading.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+The following configuration settings are available for the Open Liberty default executor pool:
+
+* `coreThreads` +
+This is the minimum number of threads in the pool (minimum 4).
+Open Liberty creates a new thread for each piece of offered work until the number of threads equals the value of this attribute.
+If `coreThreads` is not configured, it defaults to a multiple of the number of CPUs (hardware threads) available to the Open Liberty process.
+
+If Open Liberty is running in a shared environment, the thread pool controller cannot account for other processes with which it is sharing the available CPUs.
+In these cases, the default `coreThreads` value might cause Open Liberty to spin up more threads than is optimal, considering the other processes that are competing for CPU resources.
```
Comment (gjdeval) [inline ref/general/threading.adoc]: this paragraph should be indented to line up under/as-part-of the coreThreads bullet item

File: ref/general/threading.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+In this situation, you can limit `coreThreads` to a value that reflects only the proportion of the CPU resources that Open Liberty needs to run.
+
+* `maxThreads` +
+This is the maximum number of threads in the pool.
+The default value is -1, which is equal to `MAX_INT` or, effectively unlimited.
+
+In some environments, there is a hard limit on the number of threads that a process can spin up.
```
Comment (gjdeval) [inline ref/general/threading.adoc]: this paragraph should be indented to line up under the maxThreads bullet item

File: ref/general/threading.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: You can configure an LDAP user registry to manage authentication and authorization for your applications that run on Open Liberty.
```
Comment (gjdeval) [inline ref/general/threading.adoc]: wrong page description?

File: ref/general/threading.adoc [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+:page-layout: general-reference
+:seo-title: The Open Liberty auto-tuning thread pool
+:seo-description: You can configure an LDAP user registry to manage authentication and authorization for your applications that run on Open Liberty.
```
Comment (gjdeval) [inline ref/general/threading.adoc]: wrong seo-description

**Comment by dmuelle** [comment]
@gjdeval thanks for the review! I've updated the doc per your suggestions in a new commit

File: ref/general/threading.adoc [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+Hang resolution adds threads to the pool to allow the server to resume normal execution.
+Hang resolution also shortens the controller cycle duration to break out of the deadlock quickly.
+
+When the controller observes that tasks are being completed again, normal operation resumes.
+The controller cycle returns to its normal duration, and pool size is adjusted based on the usual throughput criteria.
+
+== When to tune the Open Liberty thread pool
+In most environments, configurations, and workloads, the Open Liberty thread pool does not require configuration or tuning.
+Open Liberty uses auto-tuning  to determine how many threads are needed to provide optimal server throughput.
```
Comment (Charlotte-Holt) [inline ref/general/threading.adoc]: I think there's an extra space between `auto-tuning` and `to determine`

File: ref/general/threading.adoc [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+* `maxThreads` +
+This setting specifies the maximum number of threads in the pool.
+The default value is -1, which is equal to `MAX_INT` or, effectively unlimited.
++
+In some environments, there is a hard limit on the number of threads that a process can spin up.
+Currently, Open Liberty has no way to know whether such a cap applies, or what the value is.
+If Open Liberty is running in a thread-limited environment, the operator can configure the `maxThreads` attribute to an acceptable value.
+
+The Open Liberty thread pool controller is designed to handle a wide range of workloads and configurations . There might be some edge cases where you need to adjust the `coreThreads` and `maxThreads` settings. However, try the default behavior first to make sure you actually need to make adjustments.
```
Comment (Charlotte-Holt) [inline ref/general/threading.adoc]: Extra space after `configurations`

**Comment by dmuelle** [comment]
@Charlotte-Holt thanks for reviewing- typos fixed in new commit

**Pull request merged**